src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -193,9 +195,39 @@ export class AccountDialog extends Modal {
 				for (const addedProvider of addedProviders) {
 					this.addProvider(addedProvider);
 				}
+			})
+			.then(async () => {
+				await this.addGitHubCopilotProvider();
```

Comment:
You add these on startup, but I don't see anything hooking into updating when a new login is made. Is that being handled so that users don't have to reload the window to get their Copilot account to show up?